crates/nu-command/src/filesystem/ls.rs

```diff
@@ -219,6 +225,69 @@ impl Command for Ls {
     }
 }
 
+impl Ls {
```

Comment:
These don't need to be in an `impl Ls` block, since they don't use `self` in the function signature.